app/home/route.ts

```diff
@@ -25,7 +25,7 @@ export default class Home extends Route {
 
   @action
   didTransition() {
-      const shouldShowVersionB = this.features.isEnabled(ABTesting.homePageVersionB);
+      const shouldShowVersionB = this.features.isEnabled(ABTesting.homePageHeroTextVersionB);
       const version = shouldShowVersionB ? 'versionB' : 'versionA';
```

Comment:
@saradbowman, should this use a different analytics label for this round of A/B testing?

Reply:
@saradbowman said that `versionA` and `versionB` should be okay as labels here.